controller/src/beerocks/master/db/node.cpp

```diff
@@ -343,3 +343,30 @@ bool node::link_metrics_data::add_receiver_link_metric(
     }
     return true;
 }
+
+bool node::ap_metrics_data::add_ap_metric_data(std::shared_ptr<wfa_map::tlvApMetric> ApMetricData)
+{
+    bssid                               = ApMetricData->bssid();
+    channel_utilization                 = ApMetricData->channel_utilization();
+    number_of_stas_currently_associated = ApMetricData->number_of_stas_currently_associated();
+
+    for (int i = 0; i < 3; i++) {
```

Comment:
It would feel much more natural to me to have the loop inside the conditions. Yes, that means we need to repeat the loop four times, but I think it's more appropriate.

Also we could use `std::copy(ApMetricData->estimated_service_info_field_ac_be(), ApMetricData->estimated_service_info_field_ac_be() + 3, estimated_service_info_field_ac_be)`